src/software/backend/simulation/physics/physics_robot.h

```diff
@@ -266,8 +266,12 @@ class PhysicsRobot
     // do come to a halt if no force is applied.
     // Damping is roughly calculated as v_new = v * exp(damping * t)
     // https://gamedev.stackexchange.com/questions/160047/what-does-lineardamping-mean-in-box2d
-    const double robot_linear_damping  = 0.2;
-    const double robot_angular_damping = 0.2;
+    // TODO: These values are currently increased in order for the bang-bang controller
+    // simulated in the robot firmware to behave reasonably. These values should be
+    // changed back to something around 0.2
+    // https://github.com/UBC-Thunderbots/Software/issues/1187
+    const double robot_linear_damping  = 2.0;
+    const double robot_angular_damping = 2.0;
```

Comment:
units?

Reply:
um there are none. To my knowledge Box2D does not have units for damping, it's a unitless number.

Reply:
I'm 5% sure it is c/m
Source:
https://github.com/erincatto/Box2D/issues/454
https://gamedev.stackexchange.com/questions/160047/what-does-lineardamping-mean-in-box2d

Reply:
The comment above shows the closed-form solution for how damping affects velocity